doc/secciones/04_planificacion.tex

```diff
@@ -2,6 +2,89 @@ \chapter{Planificación}
 
 \section{Metodología utilizada}
 
+\subsection{Corrector Ortográfico}
```

Comment:
De veras que lo de Las mayúsculas Arbitrarias tenéis Que tenerlo un poco en mente...

Reply:
Se me Debió Pasar cuando eliminé las anteriores Mayúsculas Arbitrarias en #26, pero ya Están Eliminadas junto a otras que he visto en 1a33751